tests/e2e/test_pvc_creation_performance.py

```diff
@@ -98,3 +105,61 @@ def test_multiple_pvc_creation_measurement_performance(
         logging.info(
             f"{number_of_pvcs} PVCs creation time took less than a 60 seconds"
         )
+
+    @pytest.mark.usefixtures(base_setup.__name__)
+    @polarion_id('OCS-1270')
+    @bugzilla('1741612')
+    def test_multiple_pvc_creation_after_deletion_performance(
+        self, teardown_factory
+    ):
+        """
+        Measuring PVC creation time of 75% of initial PVCs (120) in the same
+        rate after deleting 75% of the initial PVCs
+        """
+        initial_number_of_pvcs = 120
```

Comment:
maybe we can set the number of pvcs to be dynamic and to execute this test with different values?

Reply:
This specific test is to address a specific requirement stating the number is 120 PVCs.
We can create (not in this PR :) ) a separate test for it